src/Database/Element/Column.php

```diff
@@ -63,7 +87,7 @@ class Column
     /**
      * @param string $name name of column
      * @param string $type type of column
-     * @param array $settings - list of settings, available keys: null, default, length, decimals, signed, autoincrement, after, first
```

Comment:
If you know that some null setting exists ;-) I added comment back and add missing options.